com.onepassword.OnePassword.appdata.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.onepassword.OnePassword</id>
```

Comment:
`onepassword.com` isn't the vendor domain either. This doesn't meet the requirements.